src/cim/components/Agent/tableUtils.tsx

```diff
@@ -302,7 +302,7 @@ export const useAgentsTable = (
         canUnbindHost: canUnbindHost
           ? (host: Host) => {
               const agent = agents.find((a) => a.metadata?.uid === host.id);
-              return agent ? canUnbindHost(agent) : ['false'];
+              return agent ? canUnbindHost(agent) : [false, '' /* Only agent can be unbound */];
```

Comment:
This should not be needed since agent is optional in `canUnbindHost` and when there is no agent, it is handled there already: https://github.com/stolostron/console/pull/1095/files#diff-e336279de3b51f940b63be20165303c34785a1a90746a6bcdfea7746feaa4a19R94

Reply:
You are right in the recent flow. It's just keeping it safe for the future. Anyway, we need to handle `agent === undefined` somehow and this is one approach.

Reply:
You are right for that flow. We will need to handle `agent === undefined` anyway, so we move that logic into canUnbindHost

Reply:
Already done.